content/pages/download.html

```diff
@@ -48,7 +48,7 @@ <h3 class="widget-title">
 			    Dual licensing
                     </h3><!-- /.widget-title -->
                     <p>
-		    If you want to use t8code under a different licence than GPL, please <a href="{{ SITEURL }}/pages/contact.html">Contact us</a> to discuss possible dual licenses.
+		    If you want to use t8code under a different licence than GPL, ple to discuss possible dual licenses.
```

Comment:
This change does not look like an improvement to me. Can you explain your intentions?

Reply:
This was just a test. The link I removed produced an error message while building the website.
I thought that the CI process maybe stops due to this error and therefore the artifact link would not get added.

Reply:
Its just a draft PR, there is no review required here ;)